example/dj/urls.py

```diff
@@ -11,5 +13,7 @@
     url(r'^api/extra/$', ExtraResource.as_view())
 ]
 
-if django.get_version() < '1.9':
+if StrictVersion(django.get_version()) < StrictVersion('1.9'):
```

Comment:
1. try-except je Python idiomatic, než dělat if-else.
2. Sám si mě k tomuhle řešení nedávno naváděl.
3. Rozumím, že by to mohlo dělat problémy, pokud by to tak s tím modulem patterns udělali.
4. Je lepší napsat kód obecněji, než ho fixovat na verzi. Ale chápu, že to prostě někdy nejde.